lxd/storage/drivers/driver_cephfs_volumes.go

```diff
@@ -379,6 +379,11 @@ func (d *cephfs) UnmountVolume(vol Volume, keepBlockDev bool, op *operations.Ope
 
 // RenameVolume renames the volume and all related filesystem entries.
 func (d *cephfs) RenameVolume(vol Volume, newVolName string, op *operations.Operation) error {
+	// Defend against path traversal attacks.
+	if !shared.IsFileName(newVolName) {
```

Comment:
There is a `storagePools.ValidVolumeName()` utility function but it does not reject `..` so maybe check both?